plugins/ilert/src/components/ServicesPage/ServicesTable.tsx

```diff
@@ -0,0 +1,170 @@
+ * limitations under the License.
+ */
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+import { TableState } from '../../api';
+import { Service } from '../../types';
+import { StatusChip } from './StatusChip';
+
+import { Table, TableColumn } from '@backstage/core-components';
+import { ServiceActionsMenu } from '../Service/ServiceActionsMenu';
+import { ServiceLink } from '../Service/ServiceLink';
+
+const useStyles = makeStyles(theme => ({
+  empty: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+export const ServicesTable = ({
+  services,
+  tableState,
+  isLoading,
+  onChangePage,
+  onChangeRowsPerPage,
+  compact,
+}: {
+  services: Service[];
+  tableState: TableState;
+  isLoading: boolean;
+  setIsLoading: (isLoading: boolean) => void;
+  onChangePage: (page: number) => void;
+  onChangeRowsPerPage: (pageSize: number) => void;
+  compact?: boolean;
+}) => {
+  const classes = useStyles();
+
+  // const xsColumnStyle = {
+  //   width: '5%',
+  //   maxWidth: '5%',
+  // };
+  const smColumnStyle = {
+    width: '10%',
+    maxWidth: '10%',
+  };
+  // const mdColumnStyle = {
+  //   width: '15%',
+  //   maxWidth: '15%',
+  // };
+  // const lgColumnStyle = {
+  //   width: '20%',
+  //   maxWidth: '20%',
+  // };
+  const xlColumnStyle = {
+    width: '30%',
+    maxWidth: '30%',
+  };
+
+  const idColumn: TableColumn = {
+    title: 'ID',
+    field: 'id',
+    highlight: true,
+    cellStyle: smColumnStyle,
+    headerStyle: smColumnStyle,
+    render: rowData => <ServiceLink service={rowData as Service} />,
+  };
+  const nameColumn: TableColumn = {
+    title: 'Name',
+    field: 'name',
+    cellStyle: !compact ? xlColumnStyle : undefined,
+    headerStyle: !compact ? xlColumnStyle : undefined,
+    render: rowData => <Typography>{(rowData as Service).name}</Typography>,
+  };
+  const statusColumn: TableColumn = {
+    title: 'Status',
+    field: 'status',
+    cellStyle: smColumnStyle,
+    headerStyle: smColumnStyle,
+    render: rowData => <StatusChip service={rowData as Service} />,
+  };
+  const uptimeColumn: TableColumn = {
+    title: 'Uptime in the last 90 days',
+    field: 'uptimePercentage',
+    cellStyle: smColumnStyle,
+    headerStyle: smColumnStyle,
+    render: rowData => (
+      <Typography>
+        {(rowData as Service).uptime.uptimePercentage.p90}
+      </Typography>
+    ),
+  };
+  const actionsColumn: TableColumn = {
+    title: '',
+    field: '',
+    cellStyle: smColumnStyle,
+    headerStyle: smColumnStyle,
+    render: rowData => <ServiceActionsMenu service={rowData as Service} />,
+  };
+
+  const columns: TableColumn[] = compact
+    ? [nameColumn, statusColumn, uptimeColumn, actionsColumn]
+    : [idColumn, nameColumn, statusColumn, uptimeColumn, actionsColumn];
+  let tableStyle: React.CSSProperties = {};
+  if (compact) {
+    tableStyle = {
```

Comment:
if you want, makeStyles can work with passed-in props, so that you can get a class name that has the right (computed) properties. See example here https://github.com/backstage/backstage/blob/37388652396e53e3eff77ba50150cb1781fb2eb8/packages/core-components/src/components/Tabs/TabIcon.tsx#L40

Reply:
Good advice, as i was changing this i recognized that those styles don't apply though either. Removing it for now to stick to general table style.